docs/openapi/components/schemas/common/AgGradeInspectionDefect.yml

```diff
@@ -0,0 +1,61 @@
+$linkedData:
+  term: AgGradeInspectionDefect
+  '@id': https://w3id.org/traceability#AgGradeInspectionDefect
+title: Agriculture Grade Inspection Defect
+description: Information on a type of offsize / defect observed.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionDefect
+      - type: string
+        const:
+          - AgGradeInspectionDefect
+  offsizeDefect:
+    title: Offsize / Defect
+    description: Brief description of this offsize / defect.
+    type: string
+    $linkedData:
+      term: offsizeDefect
+      '@id': https://schema.org/description
```

Comment:
I don't suppose there is a better term definition than this?

Reply:
Each AgGradeInspectionDefect represents some type of defect that was observed on a lot during a grade inspection, and offsizeDefect property is a description of that type of defect. So, I think it fits well enough (or at least I don't see a better option on schema.org)